### DISCUSSION TITLE
Stability.AI released official SDXL update -- CosXL

### DISCUSSION THREAD

**Discussion by Rogala** [lllyasviel/Fooocus#2935] [Ideas]
[Reddit](https://www.reddit.com/r/CosXL/comments/1ctlw72/psa_stabilityai_released_official_sdxl_update/)

[Question for the developers, can we try out this version of the model?](https://sandner.art/cosine-continuous-stable-diffusion-xl-cosxl-on-stableswarmui/)

**Comment by mashb1t** [comment]
@Rogala this is currently not possible as the general pipeline would have to be extended / switched on too many lines for adding experimental model support. You can test it in Comfy though!
